includes/Story_Post_Type.php

```diff
@@ -315,6 +331,7 @@ public static function admin_enqueue_scripts( $hook ) {
 			[
 				'id'     => 'edit-story',
 				'config' => [
+					'autoSaveInterval' => AUTOSAVE_INTERVAL,
```

Comment:
Looks like this constant needs to be added to `tests/phpstan/bootstrap.php`

Reply:
Hmm, added the check for `defined('AUTOSAVE_INTERVAL')` instead since the constant is not coming from the plugin, however, will look into tests and since again once the AC-s get confirmed (there's some discussion currently as to use auto-saving for published posts or not).